docs/configuration.rst

```diff
@@ -71,6 +71,12 @@ A few relevant settings for your `Pyramid .ini file <http://docs.pylonsproject.o
         # below for more details
         pyramid_swagger.generate_resource_listing = false
 
+        # Enable/disable serving the entire dereferenced swagger schema in
+        # a single http call. This can be slow for larger schemas and will
+        # not work for infinitely recursive refs
```

Comment:
This is for refs that would have a ref property to itself.  This would lead to an infinite recursion and error out